### PR TITLE
Fix QECoverage in workbench

### LINKED ISSUE #24994: Starting QECoverage dialog crashes the workbench
### Expected behavior

The "QE Coverage" interface successfully launches in workbench

### Actual behavior

Unhandled exception generated when trying to launch the interface.

```
FrameworkManager-[Notice] Welcome to Mantid 3.13.20190304.2225
FrameworkManager-[Notice] Please cite: http://dx.doi.org/10.1016/j.nima.2014.07.029 and this release: http://dx.doi.org/10.5286/Software/Mantid
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/matplotlib/backends/backend_qt5agg.py", line 177, in __draw_idle_agg
    self.update()
RuntimeError: wrapped C/C++ object of type FigureCanvasQTAgg has been deleted
Aborted
```

### Steps to reproduce the behavior

* Start the workbench
* Click "Interfaces->Utility->QECoverage"-> :boom: 

### Platforms affected

All

### PR DESCRIPTION
**Pull request by AndreiSavici**
**To test:**
Follow the instructions and expected behaviour in the related issue.
Make sure it works on RHEL7 (it worked on Ubuntu)

Fixes #24994 
*This does not require release notes* because **bug in workbench**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: scripts/QECoverage.py [hunk 1]
```diff
@@ -192,6 +192,7 @@ def __init__(self, parent=None):
         self.axes.axhline(color='k')
         self.axes.set_xlabel(r'$|Q|$ ($\AA^{-1}$)')
         self.axes.set_ylabel('Energy Transfer (meV)')
+        self.canvas.draw()
```
Comment (peterfpeterson) [inline scripts/QECoverage.py]:
```suggestion
        self.canvas.idle_draw()
```

Comment (AndreiSavici) [reply scripts/QECoverage.py]:
```
Traceback (most recent call last):
  File "/SNS/users/3y9/Mantid/mantid/qt/applications/workbench/workbench/app/mainwindow.py", line 335, in <lambda>
    action.triggered.connect(lambda checked, script=script: self.launch_custom_gui(script))
  File "/SNS/users/3y9/Mantid/mantid/qt/applications/workbench/workbench/app/mainwindow.py", line 299, in launch_custom_gui
    executioner.execute(open(filename).read(), filename)
  File "/SNS/users/3y9/Mantid/mantid/qt/python/mantidqt/widgets/codeeditor/execution.py", line 171, in execute
    exec (code_obj, self.globals_ns, self.globals_ns)
  File "/SNS/users/3y9/Mantid/mantid/scripts/QECoverage.py", line 482, in <module>
    mainForm = QECoverageGUI()
  File "/SNS/users/3y9/Mantid/mantid/scripts/QECoverage.py", line 195, in __init__
    self.canvas.idle_draw()
AttributeError: 'FigureCanvasQTAgg' object has no attribute 'idle_draw'
```

Comment (AndreiSavici) [reply scripts/QECoverage.py]: If I use `draw_idle()` I get 
```
Traceback (most recent call last):
  File "/usr/lib64/python2.7/site-packages/matplotlib/backends/backend_qt5agg.py", line 181, in __draw_idle_agg
    self.update()
RuntimeError: wrapped C/C++ object of type FigureCanvasQTAgg has been deleted
```

**Review by peterfpeterson — APPROVED** [review]
Works with python2 on RHEL7

**Pull request merged**